toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -82,6 +82,9 @@ type ImageCustomizerParameters struct {
 
 	imageUuid    [UuidSize]byte
 	imageUuidStr string
+
+	rootHash     string
```

Comment:
At some point we will support multiple verity partitions. So, if possible, it might be good to write this code in anticipation of that.

For example:

```golang
type verityDeviceMetadata struct {
    hash string
    dataPartUuid string
    hashPartUuid string
}

type ImageCustomizerParameters struct {
    ...
    verityMetadata []verityDeviceMetadata
}
```